src/vpn-worker.ts

```diff
@@ -117,11 +114,11 @@ const worker = (instanceId: number, serviceId: number) => {
 	});
 
 	vpn.on('process:exit', (code, signal) => {
-		let msg = 'process exited';
+		let msg = `openvpn process (pid=${vpn.pid()})`;
 		if (code != null) {
-			msg = `process exited with code ${code}`;
+			msg = `${msg} exited (code=${code})`;
```

Comment:
This could also be
```suggestion
			msg += ` exited (code=${code})`;
```

Reply:
Is using `+=` better?

Reply:
Just that it could make it clearer we're appending to the message.  It also arguably might allow v8 to optimise the appending in ways it won't catch with the other, but that's just a function of how clever the optimiser is at this point in time (and this piece of code doesn't care about performance)